nuxeo-liveconnect-core/src/main/java/org/nuxeo/ecm/liveconnect/core/AbstractLiveConnectBlobProvider.java

```diff
@@ -203,7 +203,14 @@ public O getOAuth2Provider() {
 
     @Override
     public SimpleManagedBlob toBlob(LiveConnectFileInfo fileInfo) throws IOException {
```

Comment:
Could you remove `IOException`, this method is only declared in liveconnect: `LiveConnectBlobProvider`.

Reply:
Done